README.md

```diff
@@ -0,0 +1,13 @@
+![GEOS-Chem Logo](https://github.com/geoschem/geoschem.github.io/blob/main/img/GEOS-Chem_Logo_Light_Background.svg)
+
+[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.4681204.svg)](https://doi.org/10.5281/zenodo.4681204)
```

Comment:
Can we remove this DOI line? It points to an old version of GCClassic.

Reply:
@msulprizio: this is now fixed in commit 2bad907d5.